tensorflow-examples/src/main/java/org/tensorflow/model/examples/cnn/fastrcnn/FasterRcnnInception.java

```diff
@@ -269,79 +270,74 @@ public static void main(String[] params) {
                 //The given SavedModel SignatureDef input
                 feedDict.put("input_tensor", reshapeTensor);
                 //The given SavedModel MetaGraphDef key
-                Map<String, Tensor> outputTensorMap = model.function("serving_default").call(feedDict);
-                //detection_classes, detectionBoxes etc. are model output names
-                try (TFloat32 detectionClasses = (TFloat32) outputTensorMap.get("detection_classes");
-                     TFloat32 detectionBoxes = (TFloat32) outputTensorMap.get("detection_boxes");
-                     TFloat32 rawDetectionBoxes = (TFloat32) outputTensorMap.get("raw_detection_boxes");
-                     TFloat32 numDetections = (TFloat32) outputTensorMap.get("num_detections");
-                     TFloat32 detectionScores = (TFloat32) outputTensorMap.get("detection_scores");
-                     TFloat32 rawDetectionScores = (TFloat32) outputTensorMap.get("raw_detection_scores");
-                     TFloat32 detectionAnchorIndices = (TFloat32) outputTensorMap.get("detection_anchor_indices");
-                     TFloat32 detectionMulticlassScores = (TFloat32) outputTensorMap.get("detection_multiclass_scores")) {
-                    int numDetects = (int) numDetections.getFloat(0);
-                    if (numDetects > 0) {
-                        ArrayList<FloatNdArray> boxArray = new ArrayList<>();
-                        //TODO tf.image.combinedNonMaxSuppression
-                        for (int n = 0; n < numDetects; n++) {
-                            //put probability and position in outputMap
-                            float detectionScore = detectionScores.getFloat(0, n);
-                            //only include those classes with detection score greater than 0.3f
-                            if (detectionScore > 0.3f) {
-                                boxArray.add(detectionBoxes.get(0, n));
-                            }
+                Result result = model.function("serving_default").call(feedDict);
```

Comment:
This result needs to be in a try with resources so that all the outputs are closed.